src/domexpression.js

```diff
@@ -169,7 +169,10 @@ var _ = Mavo.DOMExpression = $.Class({
 		 * @return If one argument, array of matching DOMExpression objects.
 		 *         If two arguments, the matching DOMExpression object or null
 		 */
-		search: function(element, attribute) {
+        search: function (element, attribute) {
+            if (element === null) {
+                return element;
+            }
```

Comment:
It looks like something happened with the indentation here, perhaps you used spaces instead of tabs?